src/mesh/impls/bout/boutmesh.cxx

```diff
@@ -2541,6 +2560,7 @@ void BoutMesh::outputVars(Datafile &file) {
   file.add(MYG, "MYG", false);
   file.add(nx, "nx", false);
   file.add(ny, "ny", false);
+  file.add(LocalNz, "nz", false);
```

Comment:
Should we add `nz` instead of `LocalNz` here? Or is that for a separate PR?

Also, should we add `MZG` and `MZSUB`? Or also for future PR?